assets/src/edit-story/components/panels/sizePosition.js

```diff
@@ -62,12 +63,26 @@ const MIN_MAX = {
     MIN: 1,
     MAX: 1000,
   },
+  X: {
+    MIN: 0,
+    MAX: PAGE_WIDTH,
```

Comment:
Maybe these should be `1` and `PAGE_WIDTH - 1` or sth like this instead to ensure it stays on the Page? Otherwise, the element won't be visible.

Reply:
`X.MIN = 0` is good, I can place the element on canvas like that (it snaps).
`X.MAX = PAGE_WIDTH` is debatable, it will not be visible on the page (but not removed).

![image](https://user-images.githubusercontent.com/21036224/90871243-087bf980-e39b-11ea-9f15-501f11766072.png)

Reply:
@merapi Note that in usage it will actually be `MIN_MAX.X.MIN - width`, which means `0 - width` => the element will be out of the page.